testing/logging/test_fixture.py

```diff
@@ -137,3 +137,68 @@ def test_caplog_captures_for_all_stages(caplog, logging_during_setup_and_teardow
 
     # This reaches into private API, don't use this type of thing in real tests!
     assert set(caplog._item.catch_log_handlers.keys()) == {"setup", "call"}
+
+
+def test_ini_controls_global_log_level(testdir):
+    testdir.makepyfile(
+        """
+        import pytest
+        import logging
+        def test_log_level_override(request, caplog):
+            plugin = request.config.pluginmanager.getplugin('logging-plugin')
+            assert plugin.log_level == logging.INFO
+            logging.getLogger('catchlog').debug("DEBUG message won't be shown")
+            logging.getLogger('catchlog').warning("WARNING message will be shown")
+            assert 'DEBUG' not in caplog.text
+            assert 'WARNING' in caplog.text
+    """
+    )
+    testdir.makeini(
+        """
+        [pytest]
+        log_level=INFO
```

Comment:
Perhaps a better test here would be to set this to `ERROR` and assert that a `WARNING` message is not shown? Now that I think about it, this test will probably pass without the `ini` file at all which kinda defeats the point.